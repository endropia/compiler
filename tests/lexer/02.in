10 %1 &1 $a $A 10.2 10.12e 10e1 10.10.10 10e.10.e12 10.10e10