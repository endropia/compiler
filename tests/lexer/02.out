1	1	Integer	123	123
2	1	Double	123.123	123.123
3	1	Double	1.23123e+125	123.123e123
4	1	Double	1.23e+125	123e+123
5	1	Double	1.23e-121	123e-123
6	1	Double	1.23123e+125	123.123e+123
7	1	Double	1.23123e-121	123.123e-123
8	1	Double	123	123.
9	1	Double	10	1.e1
10	1	Integer	10	$a
11	1	Double	10	$a.
12	1	Double	1000	$a.e2
13	1	Double	1000	$a.e+2
14	1	Double	0.1	$a.e-2
15	1	Integer	7	&7
16	1	Double	7	&7.
17	1	Double	70	&7.e1
18	1	Double	10	$a.
18	4	Double	10	1e1
18	7	eof
